docs/release-notes/next-lima/PT-164691690-channel-mutual-close.md

```diff
@@ -0,0 +1,5 @@
+- The [```channel_close_mutual```](https://github.com/aeternity/protocol/blob/master/channels/ON-CHAIN.md#channel_close_mutual) 
+transaction can now be performed while the channel is not active on-chain (The channel solo closing sequence had been initiated and the channel is not yet closed), 
+this is a consensus breaking change available after the Lima fork.
+- The state channel FSM after the Lima fork accepts a shutdown message while the channel is being closed unilaterally.
+- If one of the peers refuses signing the closing transaction while the channel is not active then the request may timeout without killing the FSM.
```

Comment:
So this is available already before the Lima fork?

If so, strictly speaking this should have gone in `docs/release-notes/next/PT-164691690-....md `. (So the items in `docs/release-notes/next` can be aggregated in the release notes of 4.1.0, and the ones in `docs/release-notes/next-lima` will be aggregated in the release notes of 5.0.0 - I should have been more clear).